ports/nrf/Makefile

```diff
@@ -242,47 +247,45 @@ $(filter $(PY_BUILD)/../extmod/vfs_fat_%.o, $(PY_O)): COPT += -Os
 
 all: binary hex
 
-OUTPUT_FILENAME = firmware
-
 ## Create binary .bin file from the .out file
-binary: $(BUILD)/$(OUTPUT_FILENAME).bin
+binary: $(BUILD)/firmware.bin
 
-$(BUILD)/$(OUTPUT_FILENAME).bin: $(BUILD)/$(OUTPUT_FILENAME).elf
-	$(OBJCOPY) -O binary $< $@
+$(BUILD)/firmware.bin: $(BUILD)/firmware.hex
+	$(OBJCOPY) -O binary -I ihex $< $@
 
 ## Create binary .hex file from the .out file
-hex: $(BUILD)/$(OUTPUT_FILENAME).hex
+hex: $(BUILD)/firmware.hex
+
+## Join all hex files together to produce one flashable file
+$(BUILD)/firmware.hex: $(SOFTDEV_HEX) $(BUILD)/application.hex
```

Comment:
I wonder if this is always correct. Some devices like feather52 might have a static SOFTDEV_HEX pre-flashed. Could there be one alternative where this merge is not enforced?

Reply:
Good point! Of course you could just flash application.hex - but that's not really a solution.
I'll think about this.

Reply:
Is this still an issue, now that feather52 doesn't use OTA anymore?

Reply:
should not be an issue anymore.